index.html

```diff
@@ -4975,6 +4975,621 @@ <h2 id="AudioWorkerParamDescriptor">
+      <section>
+        <h2 id="AudioWorklet">
+          The <dfn>AudioWorklet</dfn> interface
+        </h2>
+        <p>
+          The AudioWorklet object allows importing user-supplied Javascript code
+          that runs on the audio rendering thread. This processing mechanism
+          ensures the aligned execution of the script code with other built-in
+          <a>AudioNode</a>s in the audio graph. The AudioWorklet object imports
+          and stores the definition of custom audio processor and then an
+          <a>AudioWorkletNode</a> object can be constructed based on the
+          processor definition. Note that the registered definition in an
+          AudioWorklet is valid for the entire global scope, therefore any
+          <a>AudioContext</a> in the same global scope can be used to create the
+          custom node.
+        </p>
+        <p>
+          An AudioWorklet object provides the capability to import module
+          scripts into its associated <a>AudioWorkletGlobalScope</a>. The user
+          agent can then create <a>AudioWorkletProcessor</a> classes registered
+          on the <a>AudioWorkletGlobalScopes</a> and construct their instances.
+          A processor object represents an audio processing module that runs on
+          the audio <a>rendering thread</a>. After the processor definition is
+          successfully stored, an instance of <a>AudioWorkletNode</a> can be
+          created in the main global scope. This is the counterpart of the
+          processor but lives on the main <a>control thread</a>. These two
+          components can communicate via message passing.
+        </p>
+        <dl title="interface AudioWorklet : Worklet" class="idl">
+          <dt>
+            Promise&lt;void&gt; import(DOMString moduleUrl)
+          </dt>
+          <dd>
+            <p>
+              Imports a module script returns a promise once it is fully loaded.
+            </p>
+          </dd>
+        </dl>
+
+        <h3>
+          Construction Procedure of AudioWorkletNode and AudioWorkletProcessor
+        </h3>
+        <p>
+          The construction procedure of <a>AudioWokletNode</a> and the
+          corresponding <a>AudioWorkletProcessor</a> object consists of several
+          coordinated interactions and the following algorithm specifies such
+          procedure. Note that these steps must be performed in an atomic
+          fashion.
+        </p>
+        <ol>
+          <li>
+            <em>In the window scope</em>: Start AudioWorkletNode constructor.
+          </li>
+          <li>
+            <em>In the AudioWorkletGlobalScope</em>: Locate a registered
+            AudioWorkletProcessor subclass based on name argument. If none is
+            found, or if subclass does not extend AudioWorkletProcessor, throw
+            an exception from the node constructor.
+            <ol>
+              <li>
+                Start registered constructor for custom processor subclass,
```

Comment:
If this start and the end (below) is meant to describe how to construct this, it might be clearer to do something like

1. Construct `Foo` as follows:
  1.  Do this
  1.  Do that

Not sure if this is what you're trying to say.